docs/pages/setup/reference/authentication.mdx

```diff
@@ -27,44 +27,143 @@ connector. There are several possible values (types) of 2FA:
+
   ```yaml
   auth_service:
     authentication:
       type: local
       second_factor: off
   ```
-  </TabItem>
-  <TabItem label="Dynamic resources (all editions)">
-  Create a file `cap.yaml`:
-  ```yaml
-  kind: cluster_auth_preference
-  metadata:
-    name: cluster-auth-preference
-  spec:
-    type: local
-    second_factor: "on"
-    webauthn:
-      rp_id: 'example.teleport.sh'
-  version: v2
-  ```
+ 
 
-  Create a resource:
+### Dynamic resource
 
-  ```code
-  $ tctl create -f cap.yaml
-  ```
-  </TabItem>
+Create a file called `cap.yaml` to define your `cluster_auth_preference`:
+
+```yaml
+kind: cluster_auth_preference
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: "on"
+  webauthn:
+    rp_id: 'example.teleport.sh'
+version: v2
+```
+
+Create the `cluster_auth_preference` resource via `tctl`:
+
+```code
+$ tctl create -f cap.yaml
+```
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+You can modify these settings using dynamic configuration resources.
+
+Log in to Teleport from your local machine so you can use the Enterprise
+edition of the `tctl` admin tool:
+
+```code
+$ tsh login --proxy=myinstance.teleport.sh
+$ tctl status
+```
+
+Create a file called `cap.yaml` to define your `cluster_auth_preference`:
+
+```yaml
+kind: cluster_auth_preference
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: "on"
+  webauthn:
+    rp_id: 'example.teleport.sh'
+version: v2
+```
+
+Create the `cluster_auth_preference` resource via `tctl`:
+
+```code
+$ tctl create -f cap.yaml
+```
+</TabItem>
 </Tabs>
-  
-<Admonition type="note">
-  SSO users can also register 2FA devices, but Teleport will not prompt them for 2FA during login. Login 2FA for SSO users should be handled by the SSO provider.
-</Admonition>
 
-## GitHub
+
+## Authentication connectors
+
+<Tabs>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+### GitHub
+
+This connector implements GitHub's OAuth 2.0 authentication flow. Please refer to GitHub's documentation on [Creating an OAuth App](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/)
+to learn how to create and register an OAuth app.
+
+Here is an example of this setting in a `cluster_auth_preference` resource:
+
+```yaml
+kind: cluster_auth_preference
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: github
+version: v2
+```
+
+See [Github OAuth 2.0](../admin/github-sso.mdx) for details on how to configure it.
+
+### SAML
+
+This connector type implements SAML authentication. It can be configured against any external identity manager like Okta or Auth0.
+
+Here is an example of this setting in a `cluster_auth_preference` resource:
+
+```yaml
+kind: cluster_auth_preference
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: saml
+version: v2
+```
+
+### OIDC
+
+Teleport implements OpenID Connect (OIDC) authentication, which is similar to SAML in principle.
```

Comment:
I don't think we need to mention "similar to SAML here."

Those who don't know the technical details probably don't care what it's similar to, and those who do know may or may not object to this.

I don't think we lose anything by just omitting this part. If we really do want to compare to some other option, I would move OIDC before SAML and after Github, and mention that OIDC is similar to OAuth (it's basically an extension to OAuth 2.0).